src/modules/partition/partition.schema.yaml

```diff
@@ -6,27 +6,33 @@ $id: https://calamares.io/schemas/partition
 additionalProperties: false
 type: object
 properties:
```

Comment:
Are you using some automated reformatting tool here? Most (all?) the YAML files in the Calamares repo use 4-spaces indents, so the indentation changes come off as a bit gratuitous. **But** if you have a nice formatting tool that follows some kind of JSON-schema standards, I'd be happy to integrate it into `ci/calamaresstyle` and call it a coding standard -- automated formatting is a good thing.